src/ray/core_worker/task_manager.h

```diff
@@ -101,11 +103,8 @@ using ExecutionSignalCallback = std::function<void(Status, int64_t)>;
 /// The API is not thread-safe.
 class ObjectRefStream {
  public:
-  ObjectRefStream(const ObjectID &generator_id)
-      : generator_id_(generator_id),
-        generator_task_id_(generator_id.TaskId()),
-        total_num_object_written_(0),
-        total_num_object_consumed_(0) {}
+  explicit ObjectRefStream(const ObjectID &generator_id)
```

Comment:
I'm not sure why we take a const ref here, why not take by value and move?

Reply:
updated